mkdocs.yml

```diff
@@ -44,6 +44,7 @@ nav:
                          - 'Packaging and Delivering Software with IPS': dev/pdf/ips-dev-guide.pdf
                          - 'Using distribution constructor': dev/distribution-constructor.md
                          - 'Existing tasks': dev/existing-tasks.md
+                - "SunRay Server on Hipster": handbook/sunray.md
```

Comment:
Please, consider moving link from main navigation page to 'Community Contributed Tutorials' with similar description.